Twig/Recommendation_Twig_Extension.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * This file is part of the EzSystemsRecommendationBundle package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributd with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Twig;
+
+use eZ\Publish\API\Repository\Repository;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Bundle\FrameworkBundle\Templating\DelegatingEngine;
+
+class Recommendation_Twig_Extension extends \Twig_Extension
+{
+    protected $template, $repository, $options;
+
+    public function __construct( DelegatingEngine $template, Repository $repository, array $options )
+    {
+        $this->template = $template;
+        $this->repository = $repository;
+        $this->options = $options;
+    }
+
+    public function getName()
+    {
+        return 'recommendation_extension';
+    }
+
+    public function getFunctions()
+    {
+        return array(
+            'yc_show_recommendations' => new \Twig_Function_Method( $this, 'yc_show_recommendations', array(
+                'is_safe' => array( 'html' )
+            ) ),
+
+            'yc_track_user' => new \Twig_Function_Method( $this, 'yc_track_user', array(
+                'is_safe' => array( 'html' )
+            ) )
+        );
+    }
+
+    public function yc_track_user( $locationId )
+    {
+        $response = new Response();
+
+        $userId = $this->options[ 'userId' ];
```

Comment:
Ain't it the opposite ? If I remember correctly, customerId is the one YooChoose expects. UserId in our Domain means an eZ Platform User.